app/services/merchant_service.rb

```diff
@@ -0,0 +1,23 @@
+class MerchantService
+  class << self
+
+    def connection
+      Faraday.new(url: 'http://localhost:3000/api/v1/')
+    end
+
+    def parse_json(response)
+      JSON.parse(response.body, symbolize_names: true)
+    end
+
+    def all_merchants
+      response = connection.get("merchants")
+      parse_json(response)
+    end
+
+    def merchant_items(id)
+      response = connection.get("merchants/#{id}/items")
+      parse_json(response)
+    end
```

Comment:
In your submission you asked if you should have created an ItemService as well. For the scope of this exercise and the two user stories provided, I don't think that's necessary.